WordPress/Tests/WP/I18nUnitTest.1.inc

```diff
@@ -151,9 +147,16 @@ really.
 EOD
 , 'my-slug' ); // OK.
 
+// @codingStandardsChangeSetting WordPress.WP.I18n text_domain my-slug,default
+__( "String default text domain.", "my-slug" ); // Ok.
+__( "String default text domain.", "default" ); // Ok.
+__( "String default text domain.", 'something' ); // Bad, text-domain mismatch.
+__( 'String default text domain.' ); // Warning, use default.
+
 // @codingStandardsChangeSetting WordPress.WP.I18n text_domain default
 __( 'String default text domain.', 'my-slug' ); // Bad because text_domain is only 'default'.
-__( 'String default text domain.', 'default' ); // Ok because domain is being explicit.
+__( 'String default text domain.', 'default' ); // Warning, text domain can be omitted.
+__( 'String default text domain.', /*explicitely set for reason. */ 'default' ); // Warning, text domain can be omitted (not auto-fixable.
```

Comment:
Typo: `explicitly` (and needs a space before it)

Reply:
Fixed